index.rst

```diff
@@ -78,11 +76,31 @@ This is the root object for XGBoost model.
 .. parsed-literal::
 
   {
-    "major_version" : 1,
-    "minor_version" : 0,
+    "version" : [1, 0],
+    "node_schema" : [
```

Comment:
`node_schema` and `node_stat_schema` goes inside `gbtree::model` if we consider #3 .